packages/core/src/baseTransfer.ts

```diff
@@ -0,0 +1,86 @@
+import type { Config } from './config/config.js';
+import type { Domainlike, EvmResource, Domain, SubstrateResource } from './types.js';
+
+export interface BaseTransferParams {
+  source: Domainlike;
+  destination: Domainlike;
+  resource: string | EvmResource | SubstrateResource;
+  sourceAddress: string;
+}
+
+export abstract class BaseTransfer {
+  protected _destination: Domain;
```

Comment:
do we need to use this `_something` style for naming attributes?

Reply:
It is because of the name conflict with a `getter` function